src/GPS/GPSProvider.cc

```diff
@@ -70,7 +85,11 @@ void GPSProvider::run()
             gpsDriver = new GPSDriverUBX(GPSDriverUBX::Interface::UART, &callbackEntry, this, &_reportGpsPos, _pReportSatInfo);
             baudrate = 0; // auto-configure
         }
-        gpsDriver->setSurveyInSpecs(_surveyInAccMeters * 10000, _surveryInDurationSecs);
+        gpsDriver->setSurveyInSpecs(_surveyInAccMeters * 10000.0f, _surveryInDurationSecs);
+
+        if (_useFixedBaseLoction) {
+            gpsDriver->setBasePosition(_fixedBaseLatitude, _fixedBaseLongitude, _fixedBaseAltitudeMeters * 10000.0f, _fixedBaseAccuracyMeters * 10000.0f);
```

Comment:
Altitude is given in meters, and accuracy in mm (one 0 too much):
`_fixedBaseAltitudeMeters, _fixedBaseAccuracyMeters * 1000.0f`

Reply:
Oops, thanks.

Yeah I was thinking about disabling the other ones. Let me look at that.

Other question: The field editors show a maximum number of decimal places. Although internally the full value is stored, visibly only 7 digits are shown after the decimal. What do you think the visible precision should me. Is 7 too few?

Reply:
Disregard decimal place dup question. Already answered,.